administrator/components/com_admin/sql/updates/mysql/5.2.0-2024-06-01.sql

```diff
@@ -0,0 +1,43 @@
+ALTER TABLE `#__modules`
+ADD COLUMN `menu_assignment` TEXT NULL DEFAULT NULL AFTER `params`;
```

Comment:
```suggestion
ALTER TABLE `#__modules` ADD COLUMN `menu_assignment` text AFTER `params` /** CAN FAIL **/;
```
Be consistent with how we do it elsewhere and use the `/** CAN FAIL **/` installer hint.

Reply:
@carlitorweb Why have you marked this review comment as resolved but have not applied my suggested change?

Reply:
You can use the "Commit suggestion" button on GitHub to apply my suggested changes.

Reply:
Was adding the installer hint in local. You also removed the `NULL DEFAULT NULL` part.

Reply:
It does not need `NULL DEFAULT NULL` for columns of type `text` because that's the default.

Reply:
In general when creating update SQL scripts look at existing scripts for the same database type if there is already one doing the same, in this case contain "ADD COLUMN", and do it in the same way as already done there.

Reply:
Understood